sql/src/main/java/io/crate/operation/udf/UDFLanguage.java

```diff
@@ -9,7 +9,7 @@
 
     public FunctionImplementation createFunctionImplementation(UserDefinedFunctionMetaData metaData) throws ScriptException;
 
-    public void validate(UserDefinedFunctionMetaData metadata) throws Exception;
+    public Boolean validate(UserDefinedFunctionMetaData metadata) throws ScriptException;
```

Comment:
why not use primitive `boolean`?

actually I would change the interface to return a nullable string
`null` means validation passed, otherwise error message is returned
that error message can be passed to exception in the caller then